app/build.gradle

```diff
@@ -46,13 +50,37 @@ android {
 }
 
 dependencies {
+    def nav_version = "2.5.2"
+    def lifecycle_version = "2.6.0-alpha01"
```

Comment:
You can add these in project's build.gradle in `ext` section

Reply:
What is the benefit of adding them in `ext` section?